### ISSUE TITLE
Issues getting Generated project built with VS2022

### ISSUE THREAD

**Issue by talran** [Synthlight/RE-Editor#4]
Previously was able to build out the project, and get everything working, but it seems like now I'm having issues with the Generated project to build. I'm able to build and get both the Generator and IDParser to build and work by reassigning the paths, but it seems like I'm getting a whole slew of errors trying to build the Generated project, and I'm not 100% sure if I've just spaced on something (13864 enums; 35813 structs) or if this is a sunbreak issue.

The error on fail is on the get parts of each struct not being able to be null, not sure if we just need to handle these or if we can allow them to be null (the project comes in with nullable -> false
![image](https://user-images.githubusercontent.com/10654373/178131762-5ab84c6d-912a-4537-a40d-2181521e0bd3.png)

**Comment by Synthlight** [comment]
The current commit isn't designed to do a full build. I've been reworking the generation to provide hierarchy support in files. One of the big issues was structs depending on others that were filtered out.

The other was trying to iteratively debug 40k+ files, which is a fucking nightmare.
So I added a whitelist feature to generation to include the shit I'm working on and nothing else.

So that's where it's at. The generator is in its ideal state, at least for managing dependencies in structs and enums, but hasn't yet been taught how to handle object fields and lists of objects. (Or generic classes.)
So it incorrectly treats those object fields as enums which isn't compilable.

In short, not working by design.

If you want to build it as it is now, run the generator with the whitelist arg. Check `Program.cs`for the exact arg. And whitelist the structs you want to edit.

Or go back in time to before this commit: https://github.com/Synthlight/MHR-Editor/commit/88f0af9b5b07d8e3547956e168d173128882d267

**Comment by talran** [comment]
Thanks, that covers it perfectly!
